### PR TITLE
[Enhancement] Refine Tensorflow related scripts

### PR DESCRIPTION
**Pull request by yiheng**
## What changes were proposed in this pull request?
1. Refine the usage of Tensorflow export graph script and export checkpoint script. Users now can input fewer parameters
2. Support export graph and variables from model files saved by ```tf.saved_model``` API
3. Update some docs

## How was this patch tested?
Manual test and Jenkins

### CODE REVIEW

**Comment by yiheng** [comment]
jenkins passed

File: scripts/dump_tf_graph.py [hunk 1]
```diff
@@ -13,22 +13,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+#
```
Comment (qiuxin2012) [inline scripts/dump_tf_graph.py]: Please also add some comments about what this script is doing.

File: scripts/export_tf_checkpoint.py [hunk 2]
```diff
@@ -13,20 +13,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+#
```
Comment (qiuxin2012) [inline scripts/export_tf_checkpoint.py]: some

**Comment by yiheng** [comment]
Meet code reivew

**Pull request merged**